citools/dump_failures.py

```diff
@@ -102,13 +108,15 @@ def dump_failures(branch, rev):
     summary_dict = summarize_failures()
 
     for manifest, tests in summary_dict.items():
-        print("\n" + manifest)
-        print()
+        if manifest in REGRESSIONS:
+            line = REGRESSIONS[manifest]
+            print("\n" + manifest + " Regressions class: " + line)
 
         for test, context in tests.items():
             print()
```

Comment:
maybe removing this blank line to make things more readable?

Reply:
okay

Reply:
![Screenshot 2022-01-05 at 6 49 50 PM](https://user-images.githubusercontent.com/31688613/148272274-192b8114-6bce-422d-9757-622237918eee.png)

here is what it looks like atm

Reply:
After stripping the spaces, this is what it looks like @jmaher

Reply:
![Screenshot 2022-01-05 at 9 22 34 PM](https://user-images.githubusercontent.com/31688613/148291404-57f6e51d-2723-4d54-a2d3-e23d3f66e7af.png)

Reply:
I like this a lot better- do you like it @chichi012 ?

Reply:
It does look better arranged/grouped. I like the single space between the manifests @jmaher

Reply:
yes, I think we can resolve this part of the patch conversation!  thanks for making this look good